src/styles/global.css

```diff
@@ -0,0 +1,67 @@
+/* || IMPORTED GOOGLE FONTS */
+
+/* || IMPORTED GOOGLE ICONS */
+.material-symbols-outlined {
+  font-variation-settings: 'FILL' 0, 'wght' 400, 'GRAD' 0, 'opsz' 24;
+}
+
+/* || GENERAL STYLES */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  /* Page Color palette */
+  --off-white: #f8f6ea;
```

Comment:
Love me some css vars! I don't know what you used to pick colors but coolers.co is a great resource.

Reply:
I ended up using the Adobe color wheel. I will will to checkout the cooler.co too :-)